altair/vegalite/v5/api.py

```diff
@@ -2678,29 +2679,29 @@ def serve(
         )
 
     def show(
-        self, embed_opt: Optional[dict] = None, open_browser: Optional[bool] = None
+        self,
+        embed_options: Optional[dict] = None,
+        using: Union[str, Iterable[str], None] = None,
+        port: Optional[int] = None,
     ) -> None:
-        """Show the chart in an external browser window.
+        """Show the chart in an external browser tab.
 
-        This requires a recent version of the altair_viewer package.
+        This requires the vl-convert-python package to be installed
 
         Parameters
         ----------
-        embed_opt : dict (optional)
+        embed_options : dict (optional)
             The Vega embed options that control the display of the chart.
-        open_browser : bool (optional)
-            Specify whether a browser window should be opened. If not specified,
-            a browser window will be opened only if the server is not already
-            connected to a browser.
+        using: str or iterable of str
+            Name of the web browser to open (e.g. "chrome", "firefox", etc.).
```

Comment:
Should we mention that it can also be a path?

Reply:
I didn't realize it could be, yeah that's useful information

Reply:
The docs don't seem to mention this: https://docs.python.org/3/library/webbrowser.html#webbrowser.register. Does this work for you?

Reply:
It does, but since it is not in the official docs, then I don't think we need to mention it either